ExampleOutput/Commands.g.cs

```diff
@@ -2,30 +2,35 @@
 using System.CommandLine;
 using System.CommandLine.Invocation;
 using System.CommandLine.Parsing;
-using DemoHandlersUpdated;
+using Jackfruit;
 
-namespace Jackfruit
+namespace DemoHandlersUpdated
 {
-    public partial class FranshiseCommand : GeneratedCommandBase<FranshiseCommand.Result>
+    public partial class CliRoot : GeneratedCommandBase<CliRoot, CliRoot.Result>
     {
-        private FranshiseCommand() : base("rootCommand")
+        public static CliRoot Create(Delegate methodToRun)
```

Comment:
The delegate is ONLY used during generation so this is to provide nice syntax in Main.